setup.py

```diff
@@ -1,13 +1,24 @@
 #!/usr/bin/env python
 
+import os
+import sys
+
 from setuptools import setup
 
 
+if sys.argv[-1] == 'publish':
+    os.system('python setup.py register sdist upload')
```

Comment:
https://pythonhosted.org/setuptools/setuptools.html#upload-upload-source-and-or-egg-distributions-to-pypi says it'll work.